service/src/main/java/cleveres/tricky/cleverestech/BackupTransaction.kt

```
@@ -100,20 +100,19 @@ internal object BackupRestoreTransaction {
                     if (existed) {
                         val copy = File(transactionDir, index.toString().padStart(4, '0') + ".bak")
-                        Files.copy(
-                            path,
-                            copy.toPath(),
-                            LinkOption.NOFOLLOW_LINKS,
-                            StandardCopyOption.COPY_ATTRIBUTES,
-                        )
-                        if (!Files.isRegularFile(copy.toPath(), LinkOption.NOFOLLOW_LINKS)) {
+                        try {
+                            Files.copy(path, copy.toPath(), StandardCopyOption.COPY_ATTRIBUTES)
+                        } catch (e: java.io.IOException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)